tests/cql/CqlDateTimeOperatorsTest.xml

```diff
@@ -580,7 +585,7 @@
 		<test name="DateTimeDurationBetweenUncertainAdd">
 			<expression>(days between DateTime(2014, 1, 15) and DateTime(2014, 2))
 				+ (days between DateTime(2014, 1, 15) and DateTime(2014, 2))</expression>
-			<output>Interval[ 34, 88 ]</output>
+			<output>Interval[ 32, 88 ]</output>
```

Comment:
See `DateTimeDurationBetweenUncertainInterval` for calculation of	`days between DateTime(2014, 1, 15) and DateTime(2014, 2)`, then:
* Uncertainty(16, 44) + Uncertainty(16, 44) = `Uncertainty(32, 44)`